Extensions/ArtifactEngine/EngineTests/artifactEngineTests.ts

```diff
@@ -80,7 +96,7 @@ describe('artifactEngine.processItems', () => {
     it('should call getArtifactItem only for artifact items that match include pattern', (done) => {
         var testProvider = new providers.StubProvider();
         var downloadOptions = new engine.ArtifactEngineOptions();
-        downloadOptions.itemPattern = 'path1\\**\npath3\\**\n!path4\\**';
+        downloadOptions.itemPattern = 'path1/**\npath3/**\n!path4/**';
```

Comment:
keep one test that has back-slash in pattern & make sure it continues to work?

Reply:
already there.